docs/guides/heroku.md

```diff
@@ -1,9 +1,132 @@
 <!--[meta]
 section: guides
-title: Deploying Keystone to Heroku
-draft: true
+title: Heroku
+subSection: deployment
 [meta]-->
 
 # Deploying Keystone to Heroku
 
-In this guide we will learn how to deploy our Keystone system to Heroku.
+This quick start guide shows how to deploy your first KeystoneJS app to Heroku and use MongoDB Atlas as our database service.
+
+## Requirements
+
+Before we start, make sure you have:
+
+- [Git](https://git-scm.com/downloads) installed with `git --version`.
+- [Docker](https://docs.docker.com/install/) installed with `docker -v`.
```

Comment:
Docker is listed here as a requirement but never mentioned in the guide. I personally don't use docker to deploy to Heroku but rather connect it to a github repo or use the CLI.

Do we need to mention docker?

Reply:
You are correct. I used both, but not in the guide. The Docker requirement is not needed.